{{cookiecutter.project_slug}}/.github/workflows/pypi.yml

```diff
@@ -1,15 +1,10 @@
 name: Build Wheels + PyPI deploy
 
 on:
-  push:
```

Comment:
is this removed because the wheel builds are too slow to always run?
it might be good to add the python tests to the main CI if the wheels CI isn't going to run by default, otherwise failing python tests might suddenly appear when a release is made.

Reply:
I considered it overkill to build all wheels, but you are right: Some sort of Python testing should be performed. It should be part of `ci.yml` though, as the `pypi.yml` workflow only exists if requested. Will work on that.